src/jimgw/transforms.py

```diff
@@ -445,3 +445,54 @@ def __init__(
             )
             for i in range(len(name_mapping[1]))
         }
+
+
+def create_bijective_transform(
```

Comment:
I think this is not a good practice in general. Here `transform_func_array` and `inverse_transform_func_array` can be potentially harmful functions, and in the long run, this flexibility will create trouble in reproducibility. Any bijective transform should be explicitly declared with the class interface.